src/redux/store.js

```diff
@@ -1,6 +1,17 @@
-import { createStore } from "redux";
-import rootReducer from "./rootReducer";
+import { createStore, combineReducers, applyMiddleware, compose } from "redux";
+import thunk from "redux-thunk";
+import authReducer from "./reducer/auth";
 
-const store = createStore(rootReducer);
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+
+const rootReducer = combineReducers({
+    authState: authReducer
+});
+
+const store = createStore(
+    rootReducer,
+    composeEnhancers(applyMiddleware(thunk))
+);
 
 export default store;
+
```

Comment:
This codes performs the same functionality with codes from `reducer/store.js` please consider removing this file `rootReducer.js` and remain with `reducer/store.js`